org.sugarlabs.Pointillism.json

```diff
@@ -0,0 +1,254 @@
+{
+    "app-id": "org.sugarlabs.Pointillism",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=all",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.Pointillism",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/Pointillism.activity/"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+
+        "shared-modules/dbus-glib/dbus-glib-0.110.json",
+        "shared-modules/intltool/intltool-0.51.json",
+         
+        "shared-modules/pygame/pygame-1.9.4.json",
+        "shared-modules/SDL/SDL-1.2.15.json",
```

Comment:
As discussed on IRC, these SDL are not necessary because `pygame-1.9.4.json` already adds them.

Reply:
Removed them, not sure if it does build correctly, make in progress